### ISSUE TITLE
Weird tower with 3dbenchy with latest master

### ISSUE THREAD

**Issue by cfelicio** [spegelius/filaswitch#24]
Not sure what happened :-)

I'm attaching the gcode, plus the processed gcode. Thanks!

![image](https://user-images.githubusercontent.com/7227683/34084931-978ec67c-e33d-11e7-93bf-2a37e5859226.png)

[3dbenchy.zip](https://github.com/spegelius/filaswitch/files/1566192/3dbenchy.zip)

**Comment by spegelius** [comment]
Hmm this can happen if the S3D has generated the g-code kind of "wrong". Please send me the gcode so I can verify. I need to update the README about how to set up S3D for dual color printing some day...

**Comment by cfelicio** [comment]
I attached the gcode as a zip file, it should contain both the original and the fs files. see above.

Oddly enough, I did not change anything on my process, but I usually move the object from the center, and this time I did not.

I tried to re-generate the gcode with the object off center, and voila, it worked! I did not change anything else besides the location of the benchy! but the gcode generated by S3D seems smaller, very odd!

 
![image](https://user-images.githubusercontent.com/7227683/34093528-9110648c-e37d-11e7-892b-d2e57a38c2a2.png)

At this point I consider this solved, but I will leave it open for now in case you want to take a look! Thanks!

[3dbenchymoved.zip](https://github.com/spegelius/filaswitch/files/1566916/3dbenchymoved.zip)
